x-pack/legacy/plugins/code/server/lsp/go_launcher.ts

```diff
@@ -104,7 +110,7 @@ export class GoServerLauncher extends AbstractLauncher {
     const params: string[] = ['-port=' + port.toString()];
     const golsp = path.resolve(this.installationPath, launchersFound[0]);
     const env = Object.create(process.env);
-    env.PATH = process.platform === 'win32' ? goHome + ';' + env.PATH : goHome + ':' + env.PATH;
+    env.PATH = gitPath + path.delimiter + goHome + path.delimiter + env.PATH;
```

Comment:
use path.resolve

Reply:
> use path.resolve

This line is used to put git binary path and go home path into the `$PATH`,  `path.delimiter` is `;` or `:`. `path.resolve` is not appropriate API here, I think?

Reply:
got it，will this works on win then？

Reply:
I have tested it on my laptop. I will test WIN later.

Reply:
Sadly, the virtual box broke on the macOS Catalina. Waiting for a stable virtual box to run the integration test.